src/client/cmd/launch.cpp

```diff
@@ -220,3 +169,111 @@ mp::ParseCode cmd::Launch::parse_args(mp::ArgParser* parser)
 
     return status;
 }
+
+mp::ReturnCode cmd::Launch::request_launch()
+{
+    mp::AnimatedSpinner spinner{cout};
+
+    auto on_success = [this, &spinner](mp::LaunchReply& reply) {
+        spinner.stop();
+
+        if (reply.metrics_pending())
+        {
+            if (mcp::is_tty())
+            {
+                cout << "Would you agree to help the Multipass developers by anonymously sending usage data?\n"
+                     << "Things we're interested in: your operating system, the images you use, the number\n"
+                     << "of instances you have, their properties and how long you use them.\n"
+                     << "We'd also like to know how long it takes for certain operations to complete.\n\n"
+                     << "Send usage data (yes/no/Later)? ";
+
+                while (true)
+                {
+                    std::string answer;
+                    std::getline(std::cin, answer);
+                    if (std::regex_match(answer, yes))
+                    {
+                        request.set_opt_in_reply(static_cast<int>(mp::OptInStatus::ACCEPTED));
```

Comment:
The RPC should have an equivalent enum and there should be to and from methods to convert between mp::OptInStatus and the RPC equivalent (therefore, removing all static_casts)